README.md

```diff
@@ -332,9 +327,10 @@ The torch wheel version `2.8.0.dev20250318+cpu` can be found at https://download
 </details>
 
 ### Docker
-
+(C++11 ABI enable by default since 2.7 release)
```

Comment:
nit: grammar: `NOTE: Since PyTorch/XLA 2.7, all builds will use the C++11 ABI by default`